sylvia/src/types.rs

```diff
@@ -1,23 +1,79 @@
+//! Module providing utilities to build and use sylvia contracts.
 use cosmwasm_std::{Deps, DepsMut, Empty, Env, MessageInfo};
 use schemars::JsonSchema;
 use serde::de::DeserializeOwned;
 use serde::{Deserialize, Serialize};
 
+/// Wrapper around [QuerierWrapper](cosmwasm_std::QuerierWrapper) for more user friendly query experience.
```

Comment:
We might add also an example of calling query method on an interface

Reply:
Good catch. Done